resources/views/feed.blade.php

```diff
@@ -22,6 +22,7 @@
             </author>
             <summary type="html">
                 <![CDATA[{!! $item->summary !!}]]>
+                <img src="{{ env('APP_URL') }}/storage/{{ $item->image }}" width="800" height="450" alt="" typeof="Image" class="image-style-body-default">
```

Comment:
This requires users to store their images in the `storage` directory (and also requiring them to set up symlinks. I think it's better to rename`image` to `imageUrl` and let the user responsible for the entire url (so remove `storage` and the env variable).